.github/workflows/ui-test.yml

```diff
@@ -29,3 +29,32 @@ jobs:
       - name: Test
         working-directory: ./ui/cap-react
         run: yarn prebuild
+
+  build-image:
+    needs: UITest
+    if: github.event_name == 'push'
+    name: Build and export ui image
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Extract branch name
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_OUTPUT
+        id: get_branch
+
+      - name: Build and export prod
+        uses: cern-sis/gh-workflows/.github/actions/docker-build@v5.5.0
```

Comment:
According to the latest changes from Ben, we have to add `cache: false` for this to work on cern registry https://cern-rcs-sis.zulipchat.com/#narrow/stream/312219-infrastructure/topic/gh-workflows/near/350702122 :)